_docs/howto/host-from-the-cloud.md

```diff
@@ -0,0 +1,23 @@
+---
+title:    Host from the Cloud
+category: Get Started
+order:    5
+---
+
+For advanced users, cloud-hosting has several advantages over [hosting from home](./host-from-home.html):
+
+ - DNS shortnames.
```

Comment:
Punctuation nitpick:

- DNS shortnames,
- Hosting via HTTPS and Dat, and
- Better reliability.